motoman_driver/MotoPlus/CtrlGroup.c

```diff
@@ -383,8 +442,8 @@ void Ros_CtrlGroup_ConvertToMotoPos(CtrlGroup* ctrlGroup, float radPos[MAX_PULSE
 	// Initilize memory space
 	memset(pulsePos, 0x00, sizeof(long)*MAX_PULSE_AXES);
 	
-	// Adjust joint order for 7 axis robot
-	if((ctrlGroup->groupId >= MP_R1_GID) && (ctrlGroup->groupId <= MP_R4_GID) && (ctrlGroup->numAxes == 7))
+	// Adjust joint order for 7 axis robot (SLEURBTE > SLURBTE)

```

Comment:
Yes, I will fix that.  However, I did not intend for this commit to be added to the PR yet.  (Another github mistake on my part.)

I'm working with someone to get a Delta robot working.  I have asked him to test this code before I try to push to the main branch.  Right now, these changes are untested.